microsoft-edge/devtools-guide-chromium/memory-problems/index.md

```diff
@@ -250,7 +250,27 @@ DevTools shows you a breakdown of memory allocation by function.  The default vi
 
 ![Allocation sampling.](../media/memory-problems-glitch-example-05-memory-allocation-sampling-heavy-bottom-up.msft.png)
 
+## Use settings for Allocation sampling to reduce garbage
```

Comment:
```suggestion

<!-- ====================================================================== -->
## Use settings for Allocation sampling to reduce garbage
```

Reply:
Maybe flip this sentence around so it starts with "Reduce garbage by ..."?

I also think that it might be worth explaining in a short sentence what reducing garbage really means and why it's important. Garbage itself is normal, it's the rate at which you're making the GC run that's a problem, right?